v2_papers/2010/lim_internet.json

```diff
@@ -0,0 +1,287 @@
+{
+  "version": "v2.1",
+  "reference": {
+    "title": "Internet Traffic Classification Demystified: On the Sources of the Discriminative Power",
+    "authors": [
+      "Lim, Yeon-sup",
+      "Kim, Hyun-chul",
+      "Jeong, Jiwoong",
+      "Kim, Chong-kwon",
+      "Kwon, Ted Taekyoung",
+      "Choi, Yanghee"
+    ],
+    "publication_name": "Co-NEXT '10 Proceedings of the 6th International COnference",
+    "publication_type": "peer_reviewed_conference",
+    "year": 2010,
+    "organization_publishers": [
+      "acm"
+    ],
+    "pages_number_of": 12,
+    "bibtex": {
+      "type": "inproceedings",
+      "booktitle": "Proceedings of the 6th International COnference",
+      "organization": "ACM"
+    },
+    "access_open": true,
```

Comment:
Unfortunately this is not open access. If you go to https://dl.acm.org/citation.cfm?id=1921180 (while outside the university network) it will ask you to pay to access the paper.